client/src/pages/CreateEnergiser.js

```diff
@@ -0,0 +1,90 @@
+import { useState } from "react";
+import Navbar from "./components/Navbar";
+import Footer from "./components/Footer";
+import "./CreateEnergiser.css";
+
+
+const CreateEnergiser = () =>{
```

Comment:
This whole function is still not indented normally - did you run your code through `prettier` that is set up in the project? Maybe someone on the team can help?